dfhack-config/init/dfhack.init

```diff
@@ -0,0 +1,11 @@
+# This file runs when DFHack is initialized, when Dwarf Fortress is first
```

Comment:
So I take it none of these files are using the `dfhack-config/default` folder where files are copied in if not present?

Reply:
They are. **Everything** in the dfhack-config source folder gets installed to dfhack-config/default

Reply:
Oh right... I figured out `data/init` installing to `hack/init` but forgot this path also changes on install